### PR TITLE
Added action to deploy to Installr (https://www.installrapp.com)

### PR DESCRIPTION
**Pull request by scottrhoyt**

### CODE REVIEW

**Comment by KrauseFx** [comment]
Please also add this action to `Actions.md`

File: lib/fastlane/actions/installr.rb [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+          builder.response :json, content_type: /\bjson$/
+          builder.use FaradayMiddleware::FollowRedirects
+          builder.adapter :net_http
+        end
+
+        options = {}
+        options[:qqfile] = Faraday::UploadIO.new(params[:ipa], 'application/octet-stream')
+
+        if params[:notes_path]
+          options[:releaseNotes] = File.read(params[:notes_path])
```
Comment (KrauseFx) [inline lib/fastlane/actions/installr.rb]: It's okay to just support `notes` without `notes_path` if you want.

**Comment by KrauseFx** [comment]
Looks great, thanks for the PR, I added some comments before it's ready to be merged

**Comment by scottrhoyt** [comment]
Thanks! I removed `notes_path` in favor of just `notes`.

**Comment by KrauseFx** [comment]
This is a great addition, thanks @scottrhoyt :rocket:

**Comment by scottrhoyt** [comment]
No problem!

**Pull request merged**